libnd4j/include/ops/declarable/helpers/cpu/compilation_units/standardDeviation.cpp.in

```diff
@@ -0,0 +1,32 @@
+/* ******************************************************************************
+ *
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ *  See the NOTICE file distributed with this work for additional
+ *  information regarding copyright ownership.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+//
+// @author AbdelRauf
+//
+
+#cmakedefine LIBND4J_TYPE_GEN 
+
+#include <ops/declarable/helpers/cpu/summaryReductions.hpp>
+
+namespace sd {
```

Comment:
Could you layout where the in files are used and how it works?

Reply:
brief story:

Reply:
@quickwritereader  just make sure to put this in the actual code itself, maybe even in a readme.

Reply:
done. also  ReadME section [AddingNewOps.md#explicit-template-instantiations-in-helper-methods](https://github.com/eclipse/deeplearning4j/blob/43a48685ba702aedc7dfdea96a49d4ea612d3022/libnd4j/AddingNewOps.md#explicit-template-instantiations-in-helper-methods)